lib/SGN/Controller/solGS/Files.pm

```diff
@@ -662,9 +663,7 @@ sub create_file_id {
 	$file_id = $sel_prop ? $file_id . '-' . $sel_prop : $file_id;
     }
 
-    my $trait_id = $traits_ids->[0];
-    $file_id = $traits_selection_id ? $file_id . '_traits_' . $traits_selection_id : $file_id . '_trait_' . $trait_id ;
-  
+    $file_id = $traits_selection_id ? $file_id . '_traits_' . $traits_selection_id : $file_id . '_trait_' . @traits_ids[0]; 
```

Comment:
i'm not sure @array[0] is proper perl, shouldn't it be $array[0]?

Reply:
ah, good catch, nick!

Reply:
fixed.

Reply:
ah, ok, accepted, merged, etc.